cli/wallet/wallet.go

```diff
@@ -332,7 +335,7 @@ func addAccount(ctx *cli.Context) error {
 }
 
 func exportKeys(ctx *cli.Context) error {
-	wall, err := openWallet(ctx.String("wallet"))
+	wall, err := readWallet(ctx.String("wallet"))
```

Comment:
We also have several other usages of `openWallet`, e.g. inside `import multisig`, `import deployed` and etc. Don't we want to replace them also?

Reply:
No, we don't, they can't be replaced because they change the wallet.